components/automate-cs-oc-erchef/habitat/config/sys.config

```diff
@@ -92,12 +92,14 @@
 
         {authz_timeout, {{cfg.authz.timeout}}},
         {authz_fanout, {{cfg.authz.fanout}}},
+        {authz_cleanup_batch_size, {{cfg.authz.cleanup_batch_size}}},
 
         {reindex_batch_size, {{cfg.index.reindex_batch_size}}},
 
         {enable_actions, false},
         {actions_fqdn, <<"{{cfg.api.actions_fqdn}}">>},
         {ldap, []},
+        {s3_url_ttl, "{{cfg.api.s3_url_ttl}}"},
```

Comment:
We should not add new config. Change in L209
`{s3_url_ttl, 28800},` to `{s3_url_ttl, {{cfg.api.s3_url_ttl}}},`